cypress-io/cypress
# chore: Add test to verify settings panels are collapsed by default

- Closes #21962

### User facing changelog


n/a

### Additional details


This PR adds a test to ensure that the panels in the app settings are collapsed by default.

See issue for details

### Steps to test


1. In `SettingsCard.vue`, add `initially-open` prop to the `Collapsible` component.
2. Run `SettingsContainer.cy.tsx` and verify that the test fails
3. Remove the prop that you added 
4. Run the tests again and verify that they pass

### How has the user experience changed?


n/a

### PR Tasks


- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
